libsnark/gadgetlib1/gadgets/hashes/anemoi/anemoi_constants.hpp

```diff
@@ -0,0 +1,264 @@
+/** @file
+ *****************************************************************************
+ * @author     This file is part of libsnark, developed by Clearmatics Ltd
+ *             (originally developed by SCIPR Lab) and contributors
+ *             (see AUTHORS).
+ * @copyright  MIT license (see LICENSE file)
+ *****************************************************************************/
+
+#ifndef LIBSNARK_GADGETLIB1_GADGETS_HASHES_ANEMOI_CONSTANTS_HPP_
+#define LIBSNARK_GADGETLIB1_GADGETS_HASHES_ANEMOI_CONSTANTS_HPP_
+
+namespace libsnark
+{
+// TODO: specialize by the field type + cast to the field
+// see     setup_sha3_constants();
+#if 0
```

Comment:
Are these required?

Reply:
Good question. These constants will be required (after proper modification) once I start implementing the round gadget. For the moment the file is just a placeholder. Would you suggest removing the whole file now and introducing it in the branch that will implement the round gadget?